spec/factories/technology.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :technology do
+    name { 'Ruby' }
```

Comment:
Not a requirement, but if you wanna be sassy you can use `Faker::ProgrammingLanguage.name`

```ruby
Faker::ProgrammingLanguage.name #=> "Ruby"
```